plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/ptf/Query.java

```diff
@@ -168,4 +171,9 @@ public ConnectorTableHandle getTableHandle()
             return tableHandle;
         }
     }
+
+    private interface ToRemoteTableName
```

Comment:
Remove this interface and change the argument of `QueryFunction` constructor to take `MongoSession`.

Reply:
I removed ToRemoteTableName Interface and inject MongoSession to QueryFunction directly.

Reply:
> Remove this interface and change the argument of `QueryFunction` constructor to take `MongoSession`.

I removed ToRemoteTableName Interface and inject MongoSession to QueryFunction directly.